packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -70,10 +70,17 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       .addClass(CHAT_MESSAGE_BOX_TEXTAREA_CLASS)
       .appendTo(this.element());
 
+    const onEnterKey = (e): void => {
```

Comment:
Why it's require to have separate variable for onEnterKey?

Reply:
Why you didn't time `e` argument?

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/28003/commits/659fc0abc38905f6de6f856a2190da2589a11f99)